src/snowflake/snowpark/session.py

```diff
@@ -2588,16 +2585,27 @@ def get_current_schema(self) -> Optional[str]:
         """
         return self._conn._get_current_parameter("schema")
 
+    @deprecated(version="1.15.0")
     def get_fully_qualified_current_schema(self) -> str:
```

Comment:
any particular reason why we are deprecating it in 1.15.0 instead of 1.14.0?

Reply:
I misunderstood the decorator 🤦 . I thought this is a "version check", and I tried to put 2 grace-period versions in and at the time we don't have 1.13.0 yet. I'll change it to 1.14.0.